### PR TITLE
Nazar/feature/add admin

### PR DESCRIPTION
**Pull request by Nazar-kob**

### CODE REVIEW

**Review by StoliarskyiKostiantyn — APPROVED** [review]

File: .github/workflows/back.yaml [hunk 1]
```diff
@@ -17,6 +17,9 @@ jobs:
         os: [ubuntu-latest]
     runs-on: ${{ matrix.os }}
     steps:
+      - uses: actions/checkout@v3.5.2
+        with:
+          submodules: true
       - uses: actions/checkout@v3
```
Comment (kipod) [inline .github/workflows/back.yaml:23]: please remove redundant step

**Review by AlexAzalor — APPROVED** [review]

File: .github/workflows/build-deploy.yaml [hunk 2]
```diff
@@ -55,6 +55,10 @@ jobs:
     runs-on: ubuntu-latest
     needs: test
     steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          submodules: true
       - uses: actions/checkout@v3
```
Comment (kipod) [inline .github/workflows/build-deploy.yaml:62]: pls remove

File: back/app/database.py [hunk 3]
```diff
@@ -5,33 +5,32 @@
...
 
 SessionLocal = sessionmaker(autocommit=False, autoflush=False, bind=engine)
 
 Base = declarative_base()
 
+db = Alchemical(DB_URI)
+
+
+class AppUser:
+    pass
```
Comment (kipod) [inline back/app/database.py:26]: ???

File: docker-compose.yaml [hunk 4]
```diff
@@ -28,6 +28,19 @@ services:
...
+  admin:
+    image: simple2b/site-s2b-admin
+    restart: always
+    command: sh ./start_server.sh
+    env_file:
+      - ${DOTENV_FILE:-project.env}
+    environment:
+      DEVEL_DATABASE_URL: postgresql://${DATABASE_USERNAME:-postgres}:${DATABASE_PASSWORD:-qwerty}@db/db
+    depends_on:
+      - back
```
Comment (kipod) [inline docker-compose.yaml:40]: back --> db

File: docker-compose.yaml [hunk 5]
```diff
@@ -28,6 +28,19 @@ services:
...
       - 127.0.0.1:${FORWARD_BACK_PORT:-5000}:80
 
+  admin:
+    image: simple2b/site-s2b-admin
+    restart: always
+    command: sh ./start_server.sh
+    env_file:
+      - ${DOTENV_FILE:-project.env}
+    environment:
+      DEVEL_DATABASE_URL: postgresql://${DATABASE_USERNAME:-postgres}:${DATABASE_PASSWORD:-qwerty}@db/db
```
Comment (kipod) [inline docker-compose.yaml:38]:
```
ENV: production
DATABASE_URL: postgresql://${DATABASE_USERNAME:-postgres}:${DATABASE_PASSWORD:-qwerty}@db/db
```

File: project.env [hunk 6]
```diff
@@ -42,3 +42,17 @@ NEXT_PUBLIC_CAPTCHA_KEY=""
...
+
+
+ADMIN_USERNAME="admin"
+ADMIN_EMAIL="admin@gmail.com"
+ADMIN_PASSWORD="admin"
+
+DEFAULT_PAGE_SIZE=8
+PAGE_LINKS_NUMBER=8
+
+FORWARD_ADMIN_PORT=5001
```
Comment (kipod) [inline project.env:58]: add EOL in EOF

**Review by kipod — APPROVED** [review]
OK: but fix my comments

**Pull request merged**